content/community-targets/ATOM_Lite_ESP32.md

```diff
@@ -0,0 +1,61 @@
+# M5Stack ATOM Lite ESP32
+
+<img src="https://cdn.shopify.com/s/files/1/0056/7689/2250/products/1_84c35eb9-97c1-4537-ba67-8ecc18ffc82d_1200x1200.jpg?v=1598833664" width="400" height="400">
```

Comment:
This image can go away at any time. Please grab it and save to the images folder.
BTW, the image for ESP32_PICO can be loaded on several devices, this is just one of them.
It's probably better to mention them. The ATOM is another good example where this images runs too.